onyo/commands/rm.py

```diff
@@ -33,11 +34,9 @@ def rm(args, opdir: str) -> None:
         print('The following will be deleted:\n' +
               '\n'.join(dryrun_list))
 
-        if not args.yes:
-            response = input('Delete assets? (y/N) ')
-            if response not in ['y', 'Y', 'yes']:
-                print('Nothing was deleted.')
-                sys.exit(0)
+        if not args.yes and not request_user_response("Save changes? No discards all changes. (y/n) "):
```

Comment:
Default to `N`.

Reply:
After discussion, Onyo will not default, and instead require an explicit `y` or `n`.